tasks/package.js

```diff
@@ -80,7 +90,8 @@ module.exports = function (grunt) {
             data: {
                 name: options.name,
                 version: options.version,
-                maintainer: options.maintainer,
+                maintainer: (options.maintainer.name ? options.maintainer.name + (options.maintainer.email ? ' <' + options.maintainer.email + '>' : '') : options.maintainer),
+                uploaders: uploadersToList(options.uploaders),
```

Comment:
Yes. First take maintainers list from package.json and put it to options. Second transform array a string.